lib/trento/users.ex

```diff
@@ -15,6 +15,14 @@ defmodule Trento.Users do
 
   alias Trento.Users.User
 
+  @spec by_id(Number.t()) :: {:ok, User.t()} | {:error, :not_found}
+  def by_id(id) do
```

Comment:
Naming suggestion: there is a convention (in this and in many other context modules) to have a verb followed by descriptive qualifier (or something along these lines; it's somewhat informal). This would mean that something like `get_by_id` or something similar might be better suited.

Reply:
I agree. The problem is that other contexts declare the `by_id` method and it's also expected by this (loosy) abstraction: https://github.com/trento-project/web/blob/b522d734e62eaedfcd598e8ce81b268296506262/lib/trento/activity_logging/parser/metadata_enricher.ex#L36

Maybe we can do a refactor afterward?